distribution/docs/src/main/resources/_contents/_configuring/configuring-from-the-admin-console-contents.adoc

```diff
@@ -0,0 +1,156 @@
+
+The ${branding} configuration can be performed through the ${admin-console} User Interface.
+
+==== Accessing the ${admin-console}
+
+Access the ${admin-console} from a web browser to configure the system.
+
+* Navigate to the ${admin-console}. (Default: \${secure_url}/admin)
+* Enter Username and Password. (Default: `admin/admin`)
+
+==== Initial Configuration
+
+The first time the ${branding} ${admin-console} runs, the initial configuration steps appear.
+
+. Click *Start* to begin.
+. On the next screen, general configuration settings such as host address, port and site name can all be configured. (See <<Configuring ${branding} Global Settings>> for important settings to configure)
+. Next, choose between a standard installation and a full installation. (Individual applications can be added, removed or deactivated later)
+
+[WARNING]
+====
+Platform App, Admin App, and Security Services App CANNOT be selected or unselected as it is installed by default and can cause errors if removed.
```

Comment:
The actual application names are `DDF Platform`, `DDF Security` and `DDF Admin`.